cli/tests/commands/schema/get.rs

```diff
@@ -0,0 +1,79 @@
+#[cfg(unix)]
+mod unix {
+    use crate::utils::{add_mock_graphql, get_cli};
+    use assert_cmd::prelude::*;
+    use predicates::prelude::*;
+    use wiremock::ResponseTemplate;
+
+    #[async_std::test]
+    async fn gets_acephi_schema() {
+        let mut cli = get_cli();
+        let response = ResponseTemplate::new(200).set_body_bytes(
+            r#"{
+                "data": {
+                    "service": {
+                        "schema": {
+                            "document": "__test__"
+                        }
+                    }
+                }
+            }"#,
+        );
+        add_mock_graphql(&mut cli, response).await.unwrap();
+
+        cli.command.arg("schema").arg("get").assert().code(0);
+    }
+
+    #[async_std::test]
+    async fn error_non_200() {
+        let mut cli = get_cli();
+        let response = ResponseTemplate::new(500);
+        add_mock_graphql(&mut cli, response).await.unwrap();
+
+        cli.command
+            .arg("schema")
+            .arg("get")
+            .assert()
+            .code(predicate::ne(0));
```

Comment:
We go through the trouble of having a custom exit code based on different errors, then let's verify we get the expected error code instead of just `ne(0)`

same in other tests.